lib/atlas/runner.rb

```diff
@@ -65,24 +50,37 @@ def runtime
       @runtime ||= Runtime.new(dataset, graph)
     end
 
-    #######
+    def graph
+      @graph ||=
+        precomputed_graph? ? dataset.graph : GraphBuilder.build
```

Comment:
This doesn't need to be split across two lines; the whole thing would be < 80 characters.